ansible/roles/pipeline-metrics/tasks/templates/openshift/pipeline-metrics-operator-repos.yaml

```diff
@@ -0,0 +1,13 @@
+---
+kind: Secret
```

Comment:
Does this need to be a secret? It looks more like configmap with repository mapping.